### PR TITLE
chore: port visual tests to /viewer

### PR DESCRIPTION
**Pull request by christjt**
~~NOTE: Depends on #2356 which should be merged first!~~

# Description

Ports all visual tests to /viewer and cleanup any references (test, CI, dependencies, test scripts etc.) from /examples.

# Checklist:

Here is a checklist that should completed before merging this given feature. 
Any shortcomings from the items below should be explained and detailed within the contents of this PR.

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the public documentation.
- [x] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [x] I have checked that my changes do not introduce regressions in the public documentation.
- [x] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [x] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [x] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
No, but there is a tracking task for it in Jira.
- [x] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [x] I have asked peers to test this feature.

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -27,51 +21,30 @@
...
     "@types/styled-components": "^5.1.4",
     "@types/three": "0.141.0",
-    "@types/webpack-env": "^1.16.2",
-    "copy-webpack-plugin": "^6.2.1",
-    "cross-env": "^7.0.2",
     "css-loader": "^6.7.1",
-    "customize-cra": "^1.0.0",
     "dotenv-webpack": "^7.1.0",
     "html-webpack-plugin": "^5.5.0",
     "jest": "^28.1.1",
```
Comment (pramodcog) [inline examples/package.json:41]: Do we use jest for examples still, I thought jest was used only for visual test in examples

Comment (christjt) [reply examples/package.json:41]: Nice catch! Nope probably not, I'll remove it.

File: viewer/packages/cad-model/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "private": true,
   "main": "index.ts",
   "scripts": {
-    "start": "yarn ws:start",
+    "start": "yarn ws:serve --env testFixture=CadClipping.VisualTest.ts",
```
Comment (pramodcog) [inline viewer/packages/cad-model/package.json:6]: Is this command that would trigger visual test in respective packages? If so, rendering package does not have the same command, how is the visual test for rendering package trigger?

**Review by pramodcog — APPROVED** [review]
LGTM!
It would be good to document `How to add new visual test and how to run it locally`

**Pull request merged**